src/main/java/edu/wisc/my/rssToJson/filter/XmlFilter.java

```diff
@@ -0,0 +1,35 @@
+package edu.wisc.my.rssToJson.filter;
+ import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+ public class XmlFilter {
+     protected final Logger logger = LoggerFactory.getLogger(getClass());
+     public static iFilter getXmlFilter(String filterName){
+        try{ 
+          String filterClass = XmlFilter.toTitleCase(filterName) + "Filter";
```

Comment:
Interesting loader here. I'm hesitant about passing user input (a piece of the path from a incoming request) directly into a function that loads arbitrary classes... But as the code is structured right now, the string wouldn't get to this line [without already being a key](https://github.com/UW-Madison-DoIT/rssToJson/pull/26/files#diff-7894a9cfeeef6526244282dd42014e39R38) in `endpoint.properties`, so I guess it's just a potential sharp edge to keep in mind in future refactors.

A way I'd feel more comfortable with would be to somehow get Spring to handle the class loading, or if we absolutely had to do it manually, get all the possible Filter class handles at startup, and load into a map with approved keys.

Reply:
Might be feasible to ask the Spring ApplicationContext for a named bean, if we want to register and dereference filter singletons that way.